### DISCUSSION TITLE
Projects (Beta) - "Convert to issue" automatically added to Github Project

### DISCUSSION THREAD

**Discussion by arthurgousset** [community/community#11961] [Projects and Issues] [answered]
Current Behaviour: 
- Issue A is part of my project
- Issue A has a list of tickboxes (e.g. `- [ ] Issue B`)
- I click "convert to issue" next to the tick box 
- A new issue is created outside the project
- I have to manually add "Issue B" to the project

Preferred behaviour: 
- Issue A is part of my project
- Issue A has a list of tickboxes (e.g. `- [ ] Issue B`)
- I click "convert to issue" next to the tick box 
- A new issue ("Issue B") is created in the same project the tickbox it was created from was in

**Comment by chriskuech** [comment]
We are trying to implement a workflow to mitigate this because we have so many issues accidentally dropped from our Project, though haven't looked to see if Projects (beta) has API support yet...

**Reply by chriskuech** [reply]
Looks like it is not...

**Comment by jwhubert91** [comment]
> Current Behaviour:
> 
> * Issue A is part of my project
> * Issue A has a list of tickboxes (e.g. `- [ ] Issue B`)
> * I click "convert to issue" next to the tick box
> * A new issue is created outside the project
> * I have to manually add "Issue B" to the project
> 
> Preferred behaviour:
> 
> * Issue A is part of my project
> * Issue A has a list of tickboxes (e.g. `- [ ] Issue B`)
> * I click "convert to issue" next to the tick box
> * A new issue ("Issue B") is created in the same project the tickbox it was created from was in

Agree that this would be desirable behavior. Entering a project, clicking into a Column and then clicking "Add new issue" should obviously go to the project and column you've clicked into. Also- if I make 20 tickets converting them all to Issue from "Draft" is extremely cumbersome. Why does draft exist at all and can we change the default to convert directly to a real issue??

**Comment by arthurgousset** [comment, marked as answer]
As of Fri, Mar 31 2023, this feature request is partially satisfied with [GitHub Issues](https://docs.github.com/en/issues) > [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects) > [Automating projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project) > [Adding items automatically](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically). It seems to have been [added on Mar 9, 2023](https://github.blog/changelog/2023-03-09-github-issues-projects-march-9th-update).

From the [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically):

> You can configure your project's built-in workflows to automatically add items from a repository that match a filter.
> 
> You can configure your project's built-in workflows to automatically add new items as they are created or updated in a repository. You can define a filter to only add items that meet your criteria.

Thanks for sharing @lvpeschke! 🙌

**Reply by mshamaseen** [reply]
This adds item from the repository to the project, what about adding items from the project to the repository issues automatically? is that supported too?

**Reply by Angelo-Verano** [reply]
I'd like to know about this too. When creating an item, I want to be able to create an issue from it automatically. I hope this becomes a feature or update.,
